src/styles/App.css

```diff
@@ -38,7 +38,7 @@
   font-weight: 900;
```

Comment:
I believe we should remove the *.css file from git, since they are autogenerated from the scss

Reply:
ok, should I add .css to the gitignore file?

Reply:
yes please